docker-compose.yml

```diff
@@ -71,6 +71,18 @@ services:
     ports:
       - "8080:8080"
 
+  # Webpack watcher
+  webpack:
```

Comment:
I'd treat this like the `oidcprovider` and include it in the `depends_on` section of the `webapp` service. That way doing:

```
docker-compose up webapp
```

brings this up, too. I tend to run the webapp this way so I don't have to deal with the extremely chatty processor.

Does that sound prudent?

Reply:
The problem is that services in `depends_on` stick around after you close the service you started with `up`, which means webpack sticks around and keeps re-compiling your frontend files, potentially with an out-of-date config (since webpack only updates the config on startup).

If there's a way to say "Start this services the webapp is up, and stop it when it is down" I'd be down for that.

Reply:
Mmm... That makes sense. We should update this documentation in some way:

https://socorro.readthedocs.io/en/latest/service/webapp.html#running-the-webapp